.eslintrc.cjs

```diff
@@ -5,7 +5,7 @@ module.exports = {
     parser: "@typescript-eslint/parser",
     project: "./tsconfig.json"
   },
-  plugins: ["@typescript-eslint", "import", "simple-import-sort"],
+  plugins: ["@typescript-eslint", "@stylistic/js", "import", "simple-import-sort"],
```

Comment:
`lines-between-class-members` got [moved](https://typescript-eslint.io/rules/lines-between-class-members/) to `@stylstic/js`.